.changes/next-release/Bug Fix-48dd227c-8804-4f2a-8f5c-54a99d1966e7.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fix setting name to reflect current functionality"
```

```suggestion
	"description": "Add new setting `aws.samcli.lambda.timeout` and remove `aws.samcli.debug.attach.timeout.millis` setting. The new setting sets the maximum time to wait for a local Lambda to start."
```

P.S.: using backticks here is nice because these changelogs are rendered in markdown in the marketplace.